### PR TITLE
Feature/less package

### PR DESCRIPTION
**Pull request by majidsajadi**
Added less loader. User options and most of the code are the same as `scss package` for easier usage.

### CODE REVIEW

**Comment by aforty** [comment]
This would be very useful to me. Can we get this vetted and merged?

**Comment by majidsajadi** [comment]
I'm waiting for any comment from maintainers then i`m ready to add tests and example.

File: packages/razzle-plugin-less/index.js [hunk 1]
```diff
@@ -1,8 +1,89 @@
...
+      prod: {
+        sourceMap: false,
+        paths: [paths.appNodeModules],
+      }
+    },
+    style: {
+      dev: {
+
+      }
+    }
```
Comment (nimaa77) [inline packages/razzle-plugin-less/index.js:40]: run prettier here

File: packages/razzle-plugin-less/package.json [hunk 2]
```diff
@@ -1,6 +1,14 @@
 {
   "name": "razzle-plugin-less",
-  "version": "2.2.0",
+  "version": "3.0.0",
+  "description": "Less package for Razzle",
```
Comment (nimaa77) [inline packages/razzle-plugin-less/package.json:4]:
```suggestion
  "description": "Use Less with Razzle",
```

File: packages/razzle-plugin-less/index.js [hunk 3]
```diff
@@ -1,8 +1,89 @@
...
+      options.less.dev :
+      options.less.prod
+  };
+
+  config.module.rules = [
+    ...config.module.rules,
+    {
+      test: /\.less$/,
+      use:
+        target === 'node'
```
Comment (nimaa77) [inline packages/razzle-plugin-less/index.js:72]: create a variable called `isServer` and use that ...

**Review by nimaa77 — COMMENTED** [review]
It's good, I don't know how Less works but don't we need `postcss-loader ` and `resolve-url-loader`?

**Comment by nimaa77** [comment]
I'm going to close this in favor of #1150

**Pull request closed without merging**